pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -113,10 +113,11 @@ type GrpcPodConfig struct {
 	// +optional
 	PriorityClassName *string `json:"priorityClassName,omitempty"`
 
-	// RunAsRoot allows admins to indicate that they wish to run the container inside the CatalogSource pod in a privileged
-	// mode as root.  This should only be enabled when running older catalog images which could not be run as non-root.
+	// RequirePrivilegeAccess when set to true, allows the containers in the pod to run in privileged mode.
+	// This should only be set to true when using older catalog images, which could not be run in restricted mode,
+	// and require privileged access.
 	// +optional
-	RunAsRoot bool `json:"runAsRoot,omitempty"`
+	RequirePrivilegedAccess bool `json:"runAsRoot,omitempty"`
```

Comment:
Should the json tag be `RequirePrivilegedAccess` as well?

Reply:
oof nice catch, thanks @dinhxuanvu!